j2objc-gradle

```diff
@@ -0,0 +1 @@
+Subproject commit 4961b7e30f7a2a5f1af3805bedcfc2e4487ce0ab
```

Comment:
That's just a visualization.  Basically in your repo you do:
`pushd j2objc-gradle && git pull && popd && git add . && git commit`

That will update the submodule to HEAD for the upstream j2objc-gradle repo.  Then it will submit a commit to this e2e test repo that says the SHA has been updated.